terraform/deployments/apps/router/main.tf

```diff
@@ -24,10 +24,11 @@ provider "aws" {
 
 module "task_definition" {
   source             = "../../../modules/task-definitions/router"
+  service_name       = "router"
   image_tag          = var.image_tag
   mesh_name          = var.mesh_name
   execution_role_arn = data.aws_iam_role.execution.arn
-  mongodb_host       = var.router_mongodb_host
+  mongodb_url        = var.router_mongodb_url
```

Comment:
Ideally this would be a list of hosts and we'd compose the URL at some appropriate point further down (probably inside the app module) - but don't worry about it for now. Just thought I should mention since you changed it. I'm guessing you changed it because you wanted to append the database name `/router`? If so, a cleaner way to do that would be to either introduce a variable for the database name or have it use `service_name` (which would let it handle multiple environments in the same account without any extra complexity).